tests/integration/use_cases/test_project.py

```diff
@@ -25,6 +26,24 @@ def test_when_create_project_it_works(kili_api_gateway: KiliAPIGateway):
     assert project_id == "fake_project_id"
 
 
+def test_when_create_project_with_project_id_it_works(kili_api_gateway: KiliAPIGateway):
+    kili_api_gateway.create_project.return_value = "fake_copied_project_id"
+
+    # When
+    project_id = ProjectUseCases(kili_api_gateway).create_project(
+        input_type="TEXT",
+        json_interface={},
+        title="test",
+        description="description",
+        project_id=ProjectId("fake_project_id"),
+        project_type=None,
+        compliance_tags=None,
+    )
+
+    # Then
+    assert project_id == "fake_copied_project_id"
```

Comment:
I'm not sure about this test, what are we testing ?

Reply:
That the id is not the same as the projectId passed as parameters, but maybe it's useless